<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="status_con">
      <property name="text">
       <string>Отключено</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0" colspan="2">
        <widget class="QDateEdit" name="dateEdit">
         <property name="maximumDateTime">
          <datetime>
           <hour>17</hour>
           <minute>55</minute>
           <second>0</second>
           <year>2017</year>
           <month>9</month>
           <day>14</day>
          </datetime>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>23</hour>
           <minute>45</minute>
           <second>0</second>
           <year>2016</year>
           <month>8</month>
           <day>15</day>
          </datetime>
         </property>
         <property name="minimumDate">
          <date>
           <year>2016</year>
           <month>8</month>
           <day>15</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="combo_airports"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="arrival_btn">
         <property name="text">
          <string>Прилет</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="flight_btn">
         <property name="text">
          <string>Вылет</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="airport">
         <property name="text">
          <string>Аэропорт</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="get_result_btn">
         <property name="text">
          <string>Получить рейсы</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QPushButton" name="workload_btn">
      <property name="text">
       <string>Отобразить загруженность</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
